tensorflow/core/util/mkl_util.h

```diff
@@ -2155,6 +2155,17 @@ inline bool IsConv1x1StrideNot1(memory::dims filter_dims,
           ((strides[0] != 1) || (strides[1] != 1)));
 }
 
+#ifdef ENABLE_MKLDNN_V1
+void execute_primitives(
+    std::vector<mkldnn::primitive>& primitives, std::shared_ptr<stream> stream,
+    std::vector<std::unordered_map<int, memory>>& net_args) {
+  DCHECK_EQ(primitives.size(), net_args.size();
```

Comment:
@nammbash line 2162 does have a closing parenthesis! It will not build.

Reply:
Thanks for the heads up! I'll fix this, along with the remaining formatting error myself then.

Reply:
@penpornk thank you let me know if you still need me to do something. There is a version mismatch clearly..which I will resolve internally.